servers/src/main/java/tachyon/worker/netty/DataServerHandler.java

```diff
@@ -118,7 +117,7 @@ private void handleBlockRequest(final ChannelHandlerContext ctx, final RPCBlockR
       final long readLength = returnLength(offset, len, fileLength);
       ChannelFuture future =
           ctx.writeAndFlush(new RPCBlockResponse(blockId, offset, readLength, getDataBuffer(req,
-              reader, readLength)));
+              reader, readLength), RPCResponse.Status.SUCCESS));
```

Comment:
Could the style be changed like the other `writeAndFlush`s? (Create variable for resp and then flush resp)